### PR TITLE
Updated CI to Xcode 12.1

### PR DESCRIPTION
**Pull request by roger-tan**
#### What's in this PR?

- Updated CI to Xcode 12.1
- Updated Main.storyboard to remove an error in Other screen
- Fixed some Carthage issues for Xcode 12.1

---

#### Pre-merge checklist

Before merging any PR, please check the following common things that should be done beforehand. These aren't all always required, so just check the box if it doesn't apply.

- [x] **When adding files, make sure they're added to the right target**. If you're adding new files that should be bundled up with Cocoapods etc, they need to be added to the `TABTestKit` target, not `Pods-TABTestKit_Example` etc.
- [x] **Run `pod install` to ensure that the latest changes are in the Example project**. Without this, Carthage might not see the latest changes.
- [x] **Added and updated tests where possible**. This isn't always possible but try wherever you can. The example app contains UI tests to test many of the TABTestKit features.
- [x] **Updated the `CHANGELOG`**. For any changes pending a release, add to the Pending section. For releases, move everything pending to the release section.
- [x] **Updated the `README`**. Add info for any new features, update existing info for anything that's changed or needs extra info.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -70,10 +94,14 @@ jobs:
...
-      - name: Set Xcode version to 11.6
-        run: sudo xcode-select -switch /Applications/Xcode_11.6.app
+      - name: Set Xcode version to 11.7
+        run: sudo xcode-select -switch /Applications/Xcode_11.7.app
       - name: Build Carthage
         run: carthage build --archive
+      - name: Set Xcode version to 12.1
+        run: sudo xcode-select -switch /Applications/Xcode_12.1.app
+      - name: Build Carthage
+        run: ./carthage.sh build --archive
```
Comment (roger-tan) [inline .github/workflows/ci.yml:104]: For Xcode 12.x, there is a workaround in order to build.

See https://github.com/Carthage/Carthage/issues/3019#issuecomment-665136323

File: Example/TABTestKit/OtherElementsController.swift [hunk 2]
```diff
@@ -66,6 +66,11 @@ final class OtherElementsController: UIViewController {
     datePicker.accessibilityIdentifier = "ExampleDatePicker"
     dateTimePicker.accessibilityIdentifier = "ExampleDateTimePicker"
     countDownTimerPicker.accessibilityIdentifier = "ExampleCountDownTimerPicker"
+    [timePicker, datePicker, dateTimePicker, countDownTimerPicker]
+      .forEach {
+        NSTimeZone.resetSystemTimeZone()
+        $0?.timeZone = NSTimeZone(abbreviation: "UTC")! as TimeZone
```
[the remark this answers is not in the document]
Comment (KaneCheshire) [reply Example/TABTestKit/OtherElementsController.swift:72]: Oh I thought there was a way to force the simulator into a specific timezone?

[the remark this answers is not in the document]
Comment (KaneCheshire) [reply Example/TABTestKit/OtherElementsController.swift:72]: I'm sure on a K+C project we've added a launchEnvironment variable matching this answer to force the sim into a specific timezone: https://stackoverflow.com/a/60172985

[the remark this answers is not in the document]
Comment (KaneCheshire) [reply Example/TABTestKit/OtherElementsController.swift:72]: Yes! Found it. You can set it to this: 

`App.shared.launchEnvironment["TZ"] = "UTC"` 

Somewhere before launching the app in the example UI tests, and that will force the simulator to use UTC, so you don't need to update the app to use a specific timezone

[the remark this answers is not in the document]
Comment (roger-tan) [reply Example/TABTestKit/OtherElementsController.swift]: Thanks @KaneCheshire!

This is working.

**Review by KaneCheshire — APPROVED** [review]
Looks good :)

**Review by Gmiller290488 — APPROVED** [review]

**Pull request merged**